evm/test/NttManager.t.sol

```diff
@@ -231,6 +231,23 @@ contract TestNttManager is Test, IRateLimiterEvents {
         assertEq(nttManager.isPaused(), false);
     }
 
+    function test_pausePauserUnpauseOnlyOwner() public {
+        // transfer pauser to another address
+        address pauser = address(0x123);
+        nttManager.transferPauserCapability(pauser);
+
+        // execute from pauser context
+        vm.startPrank(pauser);
+        assertEq(nttManager.isPaused(), false);
+        nttManager.pause();
+        assertEq(nttManager.isPaused(), true);
+
+        vm.expectRevert(
+            abi.encodeWithSelector(OwnableUpgradeable.OwnableUnauthorizedAccount.selector, pauser)
+        );
+        nttManager.unpause();
+    }
```

Comment:
Nice negative check! Can you extend this test to also prank the owner and assert we can indeed unpause still?